src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,44 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_NUMBER = 10;
+    private Object[] keys;
+    private Object[] values;
+    private int size;
+    private int index;
+
+    public StorageImpl() {
+        keys = new Object[MAX_NUMBER];
+        values = new Object[MAX_NUMBER];
```

Comment:
```suggestion
        keys = (K[]) new Object[MAX_NUMBER];
        values = (V[]) new Object[MAX_NUMBER];
```
It's better if you don't have to cast objects in code